apps/site/assets/ts/schedule/components/schedule-finder/UpcomingDepartures.tsx

```diff
@@ -139,7 +139,30 @@ export const UpcomingDepartures = ({
   if (journeys !== null && hasPredictions(journeys)) {
     return (
       <>
-        <h3>Upcoming Departures</h3>
+        <div className="schedule-table__upcoming-departures-header">
+          <h3>Upcoming Departures</h3>
+          &nbsp;
```

Comment:
Let's avoid `&nbsp;`, especially since the design doesn't call for a small space here anyway